torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19375,7 +19375,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            )),
     OpInfo('histc',
            dtypes=floating_types_and(torch.bfloat16, torch.float16),
-           dtypesIfCUDA=floating_types_and(torch.int8, torch.int16, torch.int32, torch.int64),
+           dtypesIfCUDA=floating_types_and(torch.int8, torch.uint8, torch.int16, torch.int32, torch.int64),
```

Comment:
Add `torch.uint8` to pass test
```bash
PYTORCH_OPINFO_SAMPLE_INPUT_INDEX=95 python test/test_ops.py TestCommonCUDA.test_dtypes_histc_cuda
```

Otherwise will get error like:
![image](https://github.com/user-attachments/assets/f147db19-dfe7-4ff0-a295-0cbca73a4630)


Previously the `min` and `max` value directly assign to `minvalue` and `maxvalue` will cause wrong result of comparison, which get an `RuntimeError("max must larger than min")` in test above, let test categorized `torch.uint8` as unsupported by `histc`, which is not true

![image](https://github.com/user-attachments/assets/97de58ce-a59e-4273-bb52-f8feec62d910)

![image](https://github.com/user-attachments/assets/02b00e81-536b-4fa6-bad9-148c3abd5fb2)